src/openforms/logging/models.py

```diff
@@ -107,7 +107,11 @@ def get_formatted_prefill_fields(self, fields) -> List:
 
     @property
     def fmt_prefill_fields(self) -> str:
-        if not self.extra_data or "prefill_fields" not in self.extra_data:
+        if (
+            not self.extra_data
+            or "prefill_fields" not in self.extra_data
+            or not self.content_object
```

Comment:
doesn't this result in a query for every log record?

Reply:
For every field for which `fmt_prefill_fields` is called. This happened anyway, since 2 lines later we call `self.get_formatted_prefill_fields(...)` which does:
```python
    def get_formatted_prefill_fields(self, fields) -> List:
        formatted_fields = []
        components = self.content_object.form.iter_components(recursive=True)
        ...
```